core/docs/Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2021"
 [dependencies]
```

Comment:
do we make use of the `core/docs`?

Reply:
Yes. We check if we added config options but forgot to check them in. Since there's other kinds of documentation, this could be removed, but I want the entire team to sign off on that decision.

Reply:
open an issue to discuss?

Reply:
Yep. And create a poll

Reply:
Created #1968